3p/iframe-transport-client.js

```diff
@@ -83,7 +83,20 @@ export class IframeTransportClient {
   }
 
   /**
-   * Gets the IframeMessagingClient
+   * Sends a message back to the creative.
+   * @param {string} vendor The name of the 3p vendor used in the
+   *   <amp-analytics> tag
+   * @param {string} creativeId An ID uniquely identifying which creative
+   * shall receive the event
+   * @param {!Object<string,string>} response
+   */
+  sendMessageToCreative(vendor, creativeId, response) {
```

Comment:
Since we will be adding IntersectionObserver context API here as well. I think we should rename the method here.

Reply:
Done